Trouble moving bones

Church7300 opened the pull request:
![Spring_Bone_Example](https://user-images.githubusercontent.com/106853508/174668679-7a470a31-f3de-43da-bf42-bc9f3e313676.gif)

So I used the previous version without problems but slight lag, but now it takes a while to even move one bone. it acts like it ignores click and dragging a bone to a new pose. this fixable?

schroef committed:
- Functionality to edit multiple bones at once
- Preset save mechanism
- Quick bake button
- Continous update for Animation Mode
- Show info to user in panel to go into Pose mode
- Documentation link in Preferences

Fixed
- Error when deactiving addon > wrong prop sb_collider change to sb_bone_collider

Changed
- UI to match Blender new panel design layout

schroef committed:
Correct github profile name

schroef committed:
Panel props show inactive aslong as "Spring" is not active. Helps user show it needs to be activated, also show choice of "Spring Bone" vs "Collider" object

schroef committed:
Update panel preview to match new UI adjsutments

schroef committed:
Cropped preview image correctly

artellblender committed:
Merge pull request #3 from artellblender/revert-1-master

Revert "Fix del error"

artellblender committed:
Merge pull request #2 from schroef/edit_multiple_bones

Edit multiple bones

materosx commented:
hey there sorry to bother, is this addon available still? I've seen some tutorials linking to here but it says there are no releases.
I've tried other wiggle addons but those are very incomplatible, sorry if its not the place to comment I'm just kinda frustrated that I cant get it to work and thought this one could be more compatible

artellblender committed:
Revert "Edit multiple bones"

artellblender committed:
Merge pull request #5 from artellblender/revert-2-edit_multiple_bones

Revert "Edit multiple bones"

schroef commented:
@Church7300 ,

i was the one responsible for the last upgraded version. Could i perhaps have a blend file with that armature in it?
You can take everything out which i dont need. Perhaps the mesh object would be nice so i can check the lag you are facing.

idd like to see if my edits caused this

schroef commented:
@Church7300  isnt this because that arm is a IK setup? I see the hand stays at its place and i see a yellow bone for forarm, this mostly indicates as a IK setup. Guess this causes interference with the addon and the IK setup. 

I really want to test this armature if possible

Could you also state what blender version you're using?

Likkkez committed:
Added override flags

This makes settings editable on linked characters.

Andrej730 committed:
Fix error unregistering addon

Error caused by removing non-existing property.

Example error:
```
Error: Traceback (most recent call last):
  File "\Blender\4.3\scripts\modules\addon_utils.py", line 562, in disable
    mod.unregister()
  File "\AppData\Roaming\Blender Foundation\Blender\4.3\scripts\addons\spring_bones.py", line 867, in unregister
    del bpy.types.Object.sb_collide
        ^^^^^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: type object 'Object' has no attribute 'sb_collide'
```

artellblender committed:
Merge pull request #7 from Andrej730/patch-1

Fix error unregistering addon

artellblender committed:
Merge pull request #6 from Likkkez/master

Added override flags

artellblender committed:
Updated to Blender 5x

slrgt commented:
@artellblender could we get the latest version uploaded to the official blender extension website? would be really great for discovery and future updates :) https://extensions.blender.org
